setup.py

```diff
@@ -266,7 +266,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'google-cloud-memcache>=0.2.0',
     'google-cloud-monitoring>=0.34.0,<2.0.0',
     'google-cloud-os-login>=2.0.0,<3.0.0',
-    'google-cloud-pubsub>=1.0.0,<2.0.0',
+    'google-cloud-pubsub>=2.0.0,<3.0.0',
```

Comment:
Should we change those two lines to `~2.0`  ?

Reply:
How will it benefit us? In the remaining lines, I have one convention and I think it's worth keeping one style.